service_config.py

```diff
@@ -0,0 +1,32 @@
+import os
+from logging.config import dictConfig
+
+dictConfig({
+    'version': 1,
+    'formatters': {'default': {
+        'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
+    }},
+    'handlers': {
+        'wsgi': {
+            'class': 'logging.StreamHandler',
+            'stream': 'ext://flask.logging.wsgi_errors_stream',
+            'formatter': 'default'
+        }},
+    'root': {
+        'level': 'INFO',
+        'handlers': ['wsgi']
+    }
+})
```

Comment:
For which purpose is this log config ?

Reply:
If I am not mistaken, this is a remainder of the original implementation before all our talks on the log configuration and the package. It shouldn't have any purpose at this moment. It will be purged.